qdrant-landing/content/documentation/integrations/spark.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Apache Spark
+weight: 1400
+---
+
+# Apache Spark
+
+[Spark](https://spark.apache.org/) is a leading distributed computing framework that empowers you to work with massive datasets efficiently. When it comes to leveraging the power of Spark for your data processing needs, the [Qdrant-Spark Connector](https://github.com/qdrant/qdrant-spark) is to be considered. This connector enables Qdrant to serve as a storage destination in Spark, offering a seamless bridge between the two.
+
+## Installation
+
+You can set up the Qdrant-Spark Connector in a few different ways, depending on your preferences and requirements.
+
+### GitHub Releases
+
+The simplest way to get started is by downloading pre-packaged JAR file releases from the [Qdrant-Spark GitHub releases page](https://github.com/qdrant/qdrant-spark/releases). These JAR files come with all the necessary dependencies to get you going.
+
+### Building from Source
+
+If you prefer to build the JAR from source, you'll need [JDK 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html) and [Maven](https://maven.apache.org/) installed on your system. Once you have the prerequisites in place, navigate to the project's root directory and run the following command:
+
+```bash
+mvn package -P assembly
+```
+
+This command will compile the source code and generate a fat JAR, which will be stored in the `target` directory by default.
+
+### Maven Central
+
+For Java and Scala projects, you can also obtain the Qdrant-Spark Connector from [Maven Central](https://central.sonatype.com/artifact/io.qdrant/spark).
```

Comment:
Can we add a dependency snippet for the POM here?

```xml
<dependency>
    <groupId>io.qdrant</groupId>
    <artifactId>spark</artifactId>
    <version>1.6</version>
</dependency>
```

Reply:
The collection has to be created in advance. I've emphasized the notice regarding this.
https://github.com/qdrant/landing_page/pull/389/commits/b468621c2f36308dc5fa13ba56e3dd4fb29fa282